torch/linalg/__init__.py

```diff
@@ -148,23 +148,30 @@
 """)
 
 slogdet = _add_docstr(_linalg.linalg_slogdet, r"""
-linalg.slogdet(input) -> (Tensor, Tensor)
+linalg.slogdet(input, *, out=None) -> (Tensor, Tensor)
 
 Calculates the sign and natural logarithm of the absolute value of a square matrix's determinant,
-or of the absolute values of the determinants of a batch of square matrices :attr`input`.
+or of the absolute values of the determinants of a batch of square matrices :attr:`input`.
 The determinant can be computed with ``sign * exp(logabsdet)``.
 
 Supports input of float, double, cfloat and cdouble datatypes.
 
-.. note:: When given inputs on a CUDA device, this function synchronizes that device with the CPU.
+.. note:: When given inputs on a CUDA device, this function synchronizes that device with the CPU
+          while checking if the call to MAGMA was successful using the info argument.
+
+.. note:: The determinant is computed via LU factorization. On CPU, this is done using the LAPACK
```

Comment:
"via" -> "using"

"On CPU, this is done using the LAPACK..." -> "LAPACK's \`getrf\` is used for CPU inputs, and MAGMA's \`getrf\` is used for CUDA inputs."